pymc3/tests/test_stats.py

```diff
@@ -84,9 +86,9 @@ def test_compare():
 
     traces = [trace0, trace1, trace2]
     models = [model0, model1, model2]
-    w_st = pm.compare(traces, models, method='stacking')['weight']
-    w_bb_bma = pm.compare(traces, models, method='BB-pseudo-BMA')['weight']
-    w_bma = pm.compare(traces, models, method='pseudo-BMA')['weight']
+    w_st = pm.compare(model_dict, method='stacking')['weight']
```

Comment:
missing a `model_dict = dict(zip(models, traces))` before this.

Reply:
Its up on line 73

Reply:
But these are 2 sets of different models and traces, you should repeat that line again?

Reply:
Yes, you are right. Thanks.